.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -1,60 +1,20 @@
 ---
 name: User story
-title: My user story
-about: An user story is a feature from the point of view of the end user
+about: A user story is an end-user feature
 ---
 
-<!-- This issue should be part of a PI board project -->
-
-## User
-
-<!-- Please keep the right users, and write a specific user if that's relevant. Who's that story for ? -->
-
-Study officer / Manager / Other
-
 ## Description
 
-<!-- Please give a brief description of the User Story. What must the user be able to do ? -->
-
-<!-- Is there an existing mockup or wireframe for this US ? If so, please link it here. -->
+**Who is this feature for?**
 
-## Goal
+**What should the user be able to achieve?**
 
```

Comment:
I would add this paragraph, which would be the equivalent of Goal.
`**Why is this feature necessary ?**`
If we can't answer that question that means we might be working on something pointless, so to me it is an interesting question when creating a new US.

Reply:
Maybe merge that with the `**Who is this feature for**` part?

Reply:
Hmm to me the end-user and the necessity of a new feature are two different things

Reply:
Not all features have to be strictly necessary

Reply:
I added `**Why do you think this feature would be valuable?**`